test_scripts/Polices/External_UCS/008_ATF_N_TC_LPT_Creation_PreloadedPT_with_disallowed_by_external_consent_entities_on_struct_with_invalid_type_of_params.lua

```diff
@@ -0,0 +1,145 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [Policies] External UCS: PreloadedPT with "disallowed_by_external_consent_entities_on" struct
+-- with invalid type of params
+--
+-- Description:
+-- In case:
+-- SDL uploads PreloadedPolicyTable with "disallowed_by_external_consent_entities_on:
+-- [entityType: <any_type_except_Integer>, entityId: <any_type_except_Integer>]"
+-- -> of "<functional grouping>" -> from "functional_groupings" section
+-- SDL must:
+-- a. consider this PreloadedPT as invalid
+-- b. log corresponding error internally
+-- c. shut SDL down
+--
+-- Preconditions:
+-- 1. Stop SDL
+-- 2. Modify PreloadedPolicyTable (add 'disallowed_by_external_consent_entities_on' section)
+-- Define not valid data type for entityType and entityId parameters
+--
+-- Steps:
+-- 1. Start SDL
+-- 2. Check SDL status
+--
+-- Expected result:
+-- Status = 0 (SDL is stopped)
+--
+-- Note: Script is designed for EXTERNAL_PROPRIETARY flow
+---------------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+
+--[[ Required Shared Libraries ]]
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local sdl = require('SDL')
+local testCasesForExternalUCS = require('user_modules/shared_testcases/testCasesForExternalUCS')
+
+--[[ Local variables ]]
+local grpId = "Location-1"
+local checkedSection = "disallowed_by_external_consent_entities_on"
+local params = { "entityId", "entityType" }
+local values = {
```

Comment:
@istoimenova :
1) Question "Clarification on max number of elements for disallowed_by_external_consent_entities_on / _off parameter" was raised
2) Accepted